pkgs/by-name/me/merge-ut-dictionaries/package.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+, nix-update-script
+, ruby
+, jawiki-all-titles-in-ns0
+, ibus-engines
+
+
+, mozcdic-ut-jawiki
+, mozcdic-ut-personal-names
+, mozcdic-ut-place-names
+, mozcdic-ut-sudachidict
+
+, dictionaries ? [
+    mozcdic-ut-jawiki
+    mozcdic-ut-personal-names
+    mozcdic-ut-place-names
+    mozcdic-ut-sudachidict
+  ]
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "merge-ut-dictionaries";
+  version = "0-unstable-2024-05-18";
+
+  src = fetchFromGitHub {
+    owner = "utuhiro78";
+    repo = "merge-ut-dictionaries";
+    rev = "d44c83d4bfd947fb4369e6b0c2e6111e2f9bfaba";
+    hash = "sha256-5iU44qQyYQ0e8qLsLUclrkzch6kvhbxkF/f2SrxNTHU=";
+  };
+
+  nativeBuildInputs = [ ruby ];
+
+  preConfigure =
+    if (builtins.length dictionaries == 0) then
```

Comment:
```suggestion
    if dictionaries == [] then
```

Reply:
I was trying to find out how to check if a list is empty, but that is quite simple! Thanks!